src/fides/cli/utils.py

```diff
@@ -47,6 +47,8 @@
 from fides.ctl.core.config.utils import get_config_from_file, update_config_file
 from fides.ctl.core.utils import check_response, echo_green, echo_red
 
+APP = fides.__name__
+PACKAGE = "ethyca-fides"
```

Comment:
Nice catch on these 👍🏽 do we want to just default to using the `ethyca-fides` package name for now or is there still value in the `fides` name by itself for these logs? Might be able to avoid doing `import fides` if so

Reply:
I think generally using `fides` is more accurate still for various reasons (branding, the actual name of the code module) but good thought here. Definitely something we can revisit if we feel the need in the future :)